.github/workflows/on-push-master-publish-chart.yml

```diff
@@ -0,0 +1,40 @@
+name: Publish helm chart
+
+on:
+  push:
+    branches:
+      - master
+      - main # assume that the branch name may change in future
+    paths:
+      - 'charts/**' 
+      - '.github/**'     
+env:
+  KUBE_SCORE_VERSION: 1.10.1
+  HELM_VERSION: 3.5.2
+
+jobs:
+  publish-chart:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Configure Git
+        run: |
+          git config user.name "$GITHUB_ACTOR"
+          git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
+
+      - name: Install Helm
+        uses: azure/setup-helm@v1
+        with:
+          version: v${{ env.HELM_VERSION }}
+
+      - name: Run chart-releaser
+        uses: helm/chart-releaser-action@v1.2.0
```

Comment:
I don't think we meet all https://github.com/helm/chart-releaser-action#pre-requisites?

Reply:
We'd meet them if we created a `gh-pages` branch though :)

Reply:
That was one of my questions since it's never mentioned in the PR.